src/blog/2024/08/node-red-design-patterns.md

```diff
@@ -0,0 +1,187 @@
+---
+title: "Building Better Node-RED Flows: A Comprehensive Guide to Design Patterns and Best Practices"
```

Comment:
Let's scope this down and have 600 words just about the following? Keep the rest in our pocket for further articles?
```suggestion
title: "Building Better Node-RED Flows: If Else statements and conditional logic"
```